safe/definitions/hazard_classifications.py

```diff
@@ -657,6 +657,90 @@
     'classification_unit': tr('hazard zone')
 }
 
+inundation_classes = {
+    'key': 'inundation_classes',
+    'name': tr('Inundation classes (DAM Failure)'),
+    'description': tr(
+        'This is a binary classification for an area. The area is either '
+        '<b>wet</b> (affected by flood water) or <b>dry</b> (not affected '
+        'by flood water). This unit does not describe how <b>wet</b> or '
+        '<b>dry</b> an area is.'), # ask
```

Comment:
The description(wet/dry) doesn't match the classification below with 3,2 and 1 level.
There is a comment saying you need to ask something? ;-)